test/payment/Escrow.test.js

```diff
@@ -0,0 +1,11 @@
+import shouldBehaveLikeEscrow from './Escrow.behaviour';
+
+const Escrow = artifacts.require('Escrow');
+
+contract('Escrow', function (accounts) {
+  beforeEach(async function () {
+    this.contract = await Escrow.new();
```

Comment:
Let's name this `this.escrow`, here and in all the escrow tests.

Reply:
Heh, I actually imitated the code style of the `PullPayment` tests: turns out they're pretty much the only tests not confirming to the `this.<contract_name>` style.